crates/sui/src/ptb/ptb.rs

```diff
@@ -365,6 +364,54 @@ impl PTB {
         builder.finish()
     }
 
+    // This function is used to split the input string into arguments. We decide when we have a new
+    // argument based on the different delimiters that we have. If we are inside of a delimiter and
+    // we hit a space, we should keep going until we find the end of the delimiter and a space at
+    // which point that is the end of the argument.
+    fn split_into_args(s: String) -> Vec<String> {
```

Comment:
While I know this is being tested through all the ptb test files, do you think it's a good idea to have a unit tests for this function as well?

Reply:
This is getting tested under a new `parsing` file under the tests, but yes we should make sure this is well tested, so if you spot any things that aren't tested feel free to shout

Reply:
Sounds good.